src/Controller/UserController.php

```diff
@@ -35,35 +37,43 @@ public function index()
      *
      * @Route("/users/new")
      * @Method("POST")
-     * Method will used for creating the new user from REST
+     * Method will used for creating the new user
      */
 
     public function newUserAction(Request $request)
     {
+        $educationArray = [
+            
+            'UG College' => 'UG College',
+            'PG College' => 'PG College',
+            'Masters College' => 'Masters College',
+            'Others College' => 'Others College',
+       ];
+
         $form = $this->createFormBuilder()
             ->add('userFirstName', TextType::class, array(
-                'constraints' => new NotBlank(),
+                'constraints' => new NotBlank(), 'attr' => array('class' => 'form-control')
             ))
             ->add('userLastName', TextType::class, array(
-                'constraints' => new NotBlank(),
+                'constraints' => new NotBlank(), 'attr' => array('class' => 'form-control')
             ))
             ->add('userEmail', TextType::class, array(
```

Comment:
did the requirement was to have multiple emails?

Reply:
Inthis, We can use multiple emails by using comma separator.

Reply:
Does it follows / conform to DB Normal Form / DB Normalization

Reply:
Also in general, in websites, how they accept multiple emails / mails or multiple phone numbers /  as comma separated?

Reply:
ok i will update.

Reply:
I have handled the multiple email and mobile number as expected.